tests/test_user.py

```diff
@@ -44,6 +44,52 @@ def test_get_list(app, client):
     assert res.status_code == 200
 
 
+def test_append_link(client):
+    """
+    curl -i -H "Content-Type: application/json" -H "Authorization: Bearer $x"
+    -X POST -d '{"address_name": "test.com", "categories": ["1", "2", "3"]}'
+    http://localhost:5000/user/links
+    """
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': f"Bearer {TOKEN}"
+    }
+    data = {
+        "address_name": "http://test.com",
+        "categories": ["1", "2", "3"]
+    }
+    res = client.post(CONFIG['routes']['user']['links'], headers=headers,
+                      data=json.dumps(data))
+    assert res.status_code == 200
+
+    data = {
+        "address_name": "test.com",
+        "categories": ["1", "2", "3"]
+    }
+    res = client.post(CONFIG['routes']['user']['links'], headers=headers,
+                      data=json.dumps(data))
+    assert res.status_code == 400
```

Comment:
Make separate tests for each new input with meaningful names, like:

-  linkAdding_dataContainValidURL_isSavedinDB
-  linkAdding_dataContainInvalidURL_shouldNotAccepted

Reply:
Deeply appreciate your comment, I will solve it in upcoming commit.